ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -1884,3 +1884,155 @@ def _distributed(self) -> bool:
         if self.__distributed is None:
             self.__distributed = self.parameters.numcpu > 1
         return self.__distributed
+
+    @wraps(_MapdlCore.ndinqr)
+    def ndinqr(self, node, key, **kwargs):
+        """Wrap the ``ndinqr`` method to take advantage of the gRPC methods."""
+        pname = '__tmpvar__'
+        super().ndinqr(node, key, pname=pname, mute=True, **kwargs)
```

Comment:
You can simplify the maintenance of these methods by using `*args`

Apply this here and for the remainder of these methods.

```suggestion
    def ndinqr(self, *args, **kwargs):
        """Wrap the ``ndinqr`` method to take advantage of the gRPC methods."""
        pname = '__tmpvar__'
        super().ndinqr(*args, pname=pname, mute=True, **kwargs)
```

Reply:
This makes sense. Thank you.

Reply:
It breaks the test. Hence I'm keeping the args verbosity.

However I did check that there is a workaround using an "updated wrapper"

https://docs.python.org/3/library/functools.html#functools.update_wrapper

However I felt its application wasn't very straight forward, and its advantages, little. So I did not pursue this. Let me know if I should.